cirq/ops/raw_types.py

```diff
@@ -630,6 +630,12 @@ def _is_parameterized_(self) -> bool:
             protocols.is_parameterized(tag) for tag in self.tags
         )
 
+    def _act_on_(self, args) -> bool:
```

Comment:
missing type annotation for `args`

(admittedly, this should simply be `args: Any`, but I'd still add it for consistency with other `_act_on_` definitions)

Reply:
Done.